schemas/bedrock-kms-http.js

```diff
@@ -0,0 +1,146 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const controller = {
+  title: 'controller',
+  type: 'string'
+};
+
+const referenceId = {
+  title: 'referenceId',
+  type: 'string'
+};
+
+const postKeystore = {
+  title: 'postKeystore',
+  type: 'object',
+  additionalProperties: false,
+  required: ['sequence', 'controller'],
+  properties: {
+    controller,
+    referenceId,
+    sequence: {
+      title: 'sequence',
+      type: 'number',
+      minimum: 0,
+      exclusiveMaximum: Number.MAX_SAFE_INTEGER
```

Comment:
according to this, `exclusiveMaximum` is a boolean, that works in conjuction with `maximum`

https://swagger.io/docs/specification/data-models/data-types/#range

Reply:
I was using documentation from [json-schema.org](https://json-schema.org/understanding-json-schema/reference/numeric.html#range) since that is what the bedrock-validation module uses. What specification should I follow?

Reply:
well... supposedly open API and jsonschema are *mostly* equivalent at this point.

Apparently you've hit upon one of the few differences which made this post: https://stoplight.io/blog/openapi-json-schema/

How about we side step this whole matter, don't use exclusiveMaxiumum and just set `maximum` to Number.MAX_SAFE_INTEGER - 1